projects/addon-editor/components/toolbar-new/toolbar-new.component.ts

```diff
@@ -272,3 +269,7 @@ export class TuiToolbarNewComponent {
         }
     }
 }
+
+function toolsAssertion(tools: ReadonlyArray<TuiEditorTool>): boolean {
+    return !tools.includes(TuiEditorTool.Tex) && !tools.includes(TuiEditorTool.Attach);
```

Comment:
We do not use `includes` on arrays in 2.0 since it's ES7 and we work with ES6 polyfills.

Reply:
But why then in other places of the code, these methods are used?

![image](https://user-images.githubusercontent.com/12021443/155726552-c6ae2b8a-55eb-47c5-ac9c-9912380c9dc2.png)

Reply:
`*.includes`

<img width="351" alt="image" src="https://user-images.githubusercontent.com/12021443/155727016-615cde8a-e695-44ca-9f4b-98489b8450d5.png">